uarray/_backend.py

```diff
@@ -27,6 +27,22 @@
     _SetBackendContext,
 )
 
+__all__ = [
+    "set_backend",
+    "set_global_backend",
+    "skip_backend",
+    "register_backend",
+    "clear_backends",
+    "create_multimethod",
+    "generate_multimethod",
+    "_Function",
```

Comment:
Private variables really shouldn't be in here.

Reply:
If you need to access it in tests, use `# type: ignore`

Reply:
If you do `from uarray import *` then `_Function` still isn't included. However, sometimes you really do need to access `_Function` so it helps to have it in the `uarray` module itself.

Reply:
As long as it isn't in `uarray.__all__`.